site/operational-guide/uniform-fixed-cluster.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Uniform Fixed Clusters
+menu_order: 30
+---
+> Author's Note: I'm going to change this section to recommend systemd
+> at which point the mentions of `weave launch` and `weave connect`
+> will be replaced and I will talk about amending the PEER list in
+> /etc/sysconfig/weave instead
+
+A uniform fixed cluster has the following characteristics:
+
+* Each node has identical configuration
+* There is a controlled process for adding or removing nodes. The end
+  user is responsible for ensuring that only one instance of the
+  process is in-flight at a time
+
```

Comment:
Added full stops and amended author's note to discuss reboot resilience when I revise to use systemd.